src/custom-element.js

```diff
@@ -418,6 +418,7 @@ export function createAmpElementProto(win, name, implementationClass) {
     if (this.isUpgraded()) {
       this.implementation_.layoutWidth_ = this.layoutWidth_;
     }
+    this.implementation_.onLayoutMeasure();
```

Comment:
I actually think the right thing here would be to schedule it for later call, but I added a TODO for now.